codes/plotting/fig1.R

```diff
@@ -380,14 +380,32 @@ dev.off()
 
 ## Cases Map ###
 
+# additional file to match city names
+match_city_names <- read.csv(
+  paste0(data_dir, "raw/china/match_china_city_name_w_adm2.csv"),
+  na.strings=c("", "NA"))
+
 map <- readOGR(paste0(data_dir, "interim/adm/adm2/adm2.shp"))
 map <- map[map$adm0_name == 'CHN', ]
 adm2$date <- as.Date(adm2$date, format='%Y-%m-%d')
 adm2 <- adm2[adm2$date==max(adm2$date),]
 units <- as.data.frame(map[,c("adm1_name", "adm2_name", "longitude", "latitude")])
```

Comment:
i think we dan drop this line now?

Reply:
@bolliger32 this file *is* generated manually. I did it in excel :)

Reply:
damn, impressive! I'll add a little line about this in the README under the figure 1 code section. Can we just drop the units line commented here (i know, super minor thing), and then I'll merge?

Reply:
oops you're totally right it does get used! so sorry. accepting now